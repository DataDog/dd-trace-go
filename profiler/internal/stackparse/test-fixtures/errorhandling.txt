goroutine 1 [running]:
main.main()
	/example/main.go:152 +0x3d2

goroutine 2 [running]:
main.main)(
	/example/main.go:152 +0x3d2

goroutine 3 [running]:
main.foo()
	/example/main.go:182 +0x123
created by main.main
	/example/main.go:110 +0x456
