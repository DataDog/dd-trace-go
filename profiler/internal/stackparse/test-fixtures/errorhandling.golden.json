{
  "Errors": [
    "invalid function call on line 6: \"main.main)(\""
  ],
  "Goroutines": [
    {
      "ID": 1,
      "State": "running",
      "Wait": 0,
      "LockedToThread": false,
      "Stack": [
        {
          "Func": "main.main",
          "File": "/example/main.go",
          "Line": 152
        }
      ],
      "FramesElided": false,
      "CreatedBy": null
    },
    {
      "ID": 3,
      "State": "running",
      "Wait": 0,
      "LockedToThread": false,
      "Stack": [
        {
          "Func": "main.foo",
          "File": "/example/main.go",
          "Line": 182
        }
      ],
      "FramesElided": false,
      "CreatedBy": {
        "Func": "main.main",
        "File": "/example/main.go",
        "Line": 110
      }
    }
  ]
}
